### DISCUSSION TITLE
Only 1 peer but 0 download speed

### DISCUSSION THREAD

**Discussion by thisplantguy** [transmission/transmission#4691] [Q&A]
I'm running the latest Transmission Beta for Apple Silicon, and have come across this phenomenon a few times now. Basically, if there's only 1 other peer, oftentimes I simply don't get any data downloaded.  I'd rather not share what torrent this is, but I'm attaching a screenshot of what it looks like. I'm unsure of why this happens frequently, but it seems odd that I'd have a peer but no information exchanged. 
![Screenshot 2023-01-29 at 7 34 45 PM](https://user-images.githubusercontent.com/51683498/215382554-538b0de3-472b-40bf-9da0-7eab3ff19caa.png)

Beyond hoping/waiting for more peers, is there anything I can do about this? I should mention that occasionally it will show more than 1 peer (2, 3, even 4) and the speed (or lack theirof) stays the same.

**Comment by arcticfoxwp** [comment]
@ckerr I wonder if this is #4003 as it sounds like the same issue I was having.

**Reply by thisplantguy** [reply]
@ckerr @arcticfoxwpg I'm now experiencing this phenomenon when I'm connected to multiple peers: I will be uploading, but there is no downloading occurring.
![Screenshot 2023-02-05 at 2 14 06 PM](https://user-images.githubusercontent.com/51683498/216849043-df9db4eb-f846-479f-ac51-f242c1e99a71.png)

**Reply by thisplantguy** [reply]
Furthermore, if I quit Transmission and start it up again, I begin to download again. Definitely some sort of bug in the current beta version: I'm using an M1 MacBook Air with Ventura 13.2.
